kuma/wiki/views/revision.py

```diff
@@ -84,19 +84,15 @@ def compare(request, document_slug, document_locale):
     The ids are passed as query string parameters (to and from).
     """
     locale = request.GET.get('locale', document_locale)
+    if 'from' not in request.GET or 'to' not in request.GET:
+        raise Http404
```

Comment:
I'd use ``400 Bad Request``, but others say [422 Unprocessable Entity](http://stackoverflow.com/questions/3050518/what-http-status-response-code-should-i-use-if-the-request-is-missing-a-required) and others say other values and any change would change current behavior. Maybe ``404 Not Found`` makes sense because it's a query string parameter.  Probably shouldn't change unless Roy Fielding stops by and tells us we're wrong.